### PR TITLE
Events without connexion - Mobile Optimization

### PR DESCRIPTION
**Pull request by KevinDurand974**
From issue: https://github.com/blockframes/blockframes/issues/7267

- [x] Waiting for corrected images

### CODE REVIEW

File: apps/festival/festival/src/app/event/identity/identity.component.scss [hunk 1]
```diff
@@ -1,14 +1,24 @@
...
+    background-image: url("/assets/images/light/stars_wallpaper.svg");
+    background-position: center;
+    background-repeat: no-repeat;
+    
+    section {
+      margin: auto;
+      text-align: center;
+      width: fit-content;
+    }
+  }
```
[the remark this answers is not in the document]
Comment (KevinDurand974) [reply apps/festival/festival/src/app/event/identity/identity.component.scss:21]: Yeah absolutly, but if I do this, with a content centered with flexbox, if the parent is less bigger than the child, the child will be cropped. With this the child will be never cropped.

I test these with a low resolution to see if it work, the same as Iphone SE with 50px less in height.

Simple exemple with `:host` and removing main-section:
<video src="https://user-images.githubusercontent.com/89908055/176141134-7c38ab42-97a2-4e33-a91d-930bde252149.mp4" />

With the current one:
<video src="https://user-images.githubusercontent.com/89908055/176141351-b9731914-29ec-472c-9caa-27d872dcab03.mp4" />

File: apps/festival/festival/src/app/event/identity/identity.component.html [hunk 2]
```diff
@@ -1,35 +1,37 @@
...
+  </menu>
 
-  <form fxLayout="column" [formGroup]="form" (ngSubmit)="validateIdentity()" >
+  <form [formGroup]="form" (ngSubmit)="validateIdentity()">
     <h1>Welcome to Archipel Market!</h1>
     <p class="mat-body-1">Please fill in your first and last name <br> to identify yourself.</p>
 
-    <div fxLayout fxLayoutAlign="space-between center" fxLayoutGap="16px">
-      <mat-form-field appearance="outline" fxFlex="0 1">
+    <article>
```
Comment (phpgeekfr) [inline apps/festival/festival/src/app/event/identity/identity.component.html:16]: you don't have header in `article`, `div` is not ok ?

File: styles/style.scss [hunk 3]
```diff
@@ -106,6 +106,13 @@ main {
   }
 }
 
+.surface {
+  background-color: var(--background-card);
+  padding: 24px;
+  border-radius: 4px;
+  border: 1px solid var(--foreground-divider);
```
Comment (phpgeekfr) [inline styles/style.scss:113]: css is duplicated in `styles/surface.scss`

File: apps/festival/festival/src/app/event/role/role.component.scss [hunk 4]
```diff
@@ -1,23 +1,17 @@
+@use 'surface';
+@import "@angular/flex-layout/mq";
```
Comment (phpgeekfr) [inline apps/festival/festival/src/app/event/role/role.component.scss:2]: prefer `'`

File: apps/festival/festival/src/app/event/login/login.component.scss [hunk 5]
```diff
@@ -1,37 +1,36 @@
+@use 'surface';
+@import "@angular/flex-layout/mq";
```
Comment (phpgeekfr) [inline apps/festival/festival/src/app/event/login/login.component.scss:2]:
```suggestion
@import '@angular/flex-layout/mq';
```

File: apps/festival/festival/src/app/event/identity/identity.component.scss [hunk 6]
```diff
@@ -1,34 +1,106 @@
+@use 'surface';
+@import "@angular/flex-layout/mq";
```
Comment (phpgeekfr) [inline apps/festival/festival/src/app/event/identity/identity.component.scss:2]:
```suggestion
@import '@angular/flex-layout/mq';
```

File: apps/festival/festival/src/app/event/identity/identity.component.html (hunk 2 above)
Comment (KevinDurand974) [reply apps/festival/festival/src/app/event/identity/identity.component.html:16]: We can, I think

**Review by phpgeekfr — APPROVED** [review]

File: apps/festival/festival/src/app/event/login/login.component.scss [hunk 7]
```diff
@@ -40,10 +39,61 @@
...
+        height: 48px;
+      }
+    }
+
+    article {
+      padding: 24px;
+      margin-bottom: auto;
+
+      menu {
+        margin: 0 0 12px;
```
[the remark this answers is not in the document]
Comment (KevinDurand974) [reply apps/festival/festival/src/app/event/login/login.component.scss:95]: yes

**Pull request merged**